tests/e2e/tests/regression/browser/database-list-search.e2e.ts

```diff
@@ -0,0 +1,110 @@
+import { t } from 'testcafe';
+import { acceptLicenseTerms } from '../../../helpers/database';
+import {
+    addNewStandaloneDatabases,
+    addNewOSSClusterDatabase,
+    deleteStandaloneDatabases,
+    deleteOSSClusterDatabase
+} from '../../../helpers/api/api-database';
+import { MyRedisDatabasePage } from '../../../pageObjects';
+import { rte } from '../../../helpers/constants';
+import { commonUrl, ossStandaloneConfig, ossStandaloneV5Config, ossClusterConfig } from '../../../helpers/conf';
+
+const myRedisDatabasePage = new MyRedisDatabasePage();
+const databasesForSearch = [
+    { host: ossStandaloneConfig.host, port: ossStandaloneConfig.port, databaseName: 'testSearch' },
+    { host: ossStandaloneConfig.host, port: ossStandaloneConfig.port, databaseName: 'testSecondSearch' },
+    { host: ossStandaloneV5Config.host, port: ossStandaloneV5Config.port, databaseName: 'testV5' },
+    { host: ossStandaloneConfig.host, port: ossStandaloneConfig.port, databaseName: 'lastConnection' }
+]
+const databasesForAdding = [
+    ossStandaloneConfig,
+    databasesForSearch[0],
+    databasesForSearch[1],
+    databasesForSearch[2]
+]
+
+fixture`Database list search`
+    .meta({ type: 'regression' })
+    .page(commonUrl)
+    .beforeEach(async () => {
+        // Add new databases using API
+        await acceptLicenseTerms();
+        await addNewStandaloneDatabases(databasesForAdding);
+        await addNewOSSClusterDatabase(ossClusterConfig);
+        // Reload Page
+        await t.eval(() => location.reload());
+    })
+    .afterEach(async () => {
+        //Clear and delete databases
+        await deleteStandaloneDatabases(databasesForAdding);
+        await deleteOSSClusterDatabase(ossClusterConfig);
+    });
+test
+    .meta({ rte: rte.standalone })('Verify that user can search DB by database name on the List of databases', async t => {
+        //Search for DB by name
+        const searchedDBName = 'Search'
```

Comment:
Please add `;` (don't you have eslint here? @elenaNaboko )

Reply:
We have, but for some reason VS Code doesn't highlight some issue. WebStorm does 
![image](https://user-images.githubusercontent.com/107995381/177715064-e11e1b31-a2b5-44ba-b7b7-61233f5b013f.png)